tools/driver/CMakeLists.txt

```diff
@@ -49,6 +49,9 @@ add_swift_host_tool(swift-frontend
 )
 target_link_libraries(swift-frontend
                       PUBLIC
+                        ASTGen
+                        SwiftParser
+                        SwiftSyntax
```

Comment:
Support for linking in SwiftSyntax and SwiftParser was already introduced in https://github.com/apple/swift/pull/60888. Can we ride on top of that?